cytotable/convert.py

```diff
@@ -897,13 +985,17 @@ def _to_parquet(  # pylint: disable=too-many-arguments, too-many-locals
                     "table": [
                         # perform column renaming and create potential return result
                         _prepend_column_name(
-                            # perform chunked data export to parquet using offsets
-                            table_path=_source_chunk_to_parquet(
-                                source_group_name=source_group_name,
-                                source=source,
-                                chunk_size=chunk_size,
-                                offset=offset,
-                                dest_path=dest_path,
+                            # perform data type casting work
+                            table_path=_cast_data_types(
```

Comment:
Does this make casting required? I would have thought that a more optimal implementation would be to write the parquet in a user-provided data type?

Reply:
Following up on this as a note for future reference: casting is implicitly required when moving from various data formats related to this work. When `None` is passed for the `data_type_cast_map` we use DuckDB inferenced data types to assist with CSV and SQLite compatibility towards Arrow and eventual Parquet exports.